### DISCUSSION TITLE
datadog protocol support

### DISCUSSION THREAD

**Discussion by yuanbohan** [GreptimeTeam/greptimedb#3574] [Ideas]
### What problem does the new feature solve?

support datadog protocol

### What does the feature do?

remote write datadog data into greptimedb

**Comment by tisonkun** [comment]
Move to Ideas until we have some concrete proposal.

**Comment by sunng87** [comment]
We now have a solution to use vector as a bridge for datadog and greptimedb.

![image](https://github.com/GreptimeTeam/greptimedb/assets/221942/0325b61f-ccc1-48f9-91f8-32731c53c06b)

more information:

- vector's datadog agent source: https://vector.dev/docs/reference/configuration/sources/datadog_agent/
- vector's greptimedb sink: https://vector.dev/docs/reference/configuration/sinks/greptimedb/

Just configure `DD_URL` to the address you configured for vector datadog source, and add a random `API_KEY`, it will be possible to ingest datadog agent metrics into greptimedb.
